### ISSUE TITLE
md-input-has-value class is not being applied to md-input-container when (value) changes programmatically

### ISSUE THREAD

**Issue by vladimir-ivanov** [justindujardin/ng2-material#59]
e.g.

```html
    <md-input-container>
      <label></label>
      <input md-input type="search" 
         [value]="inputValue" />
    </md-input-container>
```

seems if I type a single character and then update the value -> it works correctly, but not if I change it directly (e.g. 
  
```typescript
set value(value: string) {
    this._value = value;
    ObservableWrapper.callEmit(this.mdChange, this.value);
  }
```
does not trigger at all)

A possible fix (which works for me) is to implement OnChanges interface and force the set value method to emit for the initial case - when changes._value.previousValue is undefined - basically covering the initial state

Working code here:
https://gist.github.com/vladimir-ivanov/c45809584c1ab07eb3f3

**Comment by mcnolo** [comment]
Duplicate of #39 ?

**Comment by vladimir-ivanov** [comment]
@mcnolo - yes, better described in there, thank you, was looking for it, but could not find it as an issue.

**Comment by justindujardin** [comment]
@vladimir-ivanov This issue should be fixed in v0.2.5 which is now on NPM.  Please confirm and close.

**Comment by justindujardin** [comment]
This should be fixed in v0.2.4
